tests/ordered_db.py

```diff
@@ -1,57 +1,40 @@
 #!/usr/bin/env python3
+
 """
-# Copyright (C) 2016
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License (version 3+) as
-# published by the Free Software Foundation. You should have received
-# a copy of the GNU General Public License along with this program.
-# If not, see <http://www.gnu.org/licenses/>.
+Copyright (C) 2019 Numix Project
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License (version 3+) as
+published by the Free Software Foundation. You should have received
+a copy of the GNU General Public License along with this program.
+If not, see <http://www.gnu.org/licenses/>.
 """
 
-from os import path
-import json
 from collections import OrderedDict
+from json import load
+from os import path
 
-from utils import sort, error
+from utils import error, sort, sort_check
```

Comment:
The code for checking whether a list was sorted was included three times separately in this script rather than being turned into a function. It's now done as part of `utils.py`.